src/daniil_efremov_ex/ex04/myrand.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Daniil Efremov'
+__email__ = 'daniil.vitalevich.efremov@nmbu.no'
+
+
+class LCGRand:
+    def __init__(self, seed):
+        self.seed = seed
+
+    def rand(self):
+        a = 16807
+        m = 2 ** 31 - 1
```

Comment:
Instead of recalculating these numbers everytime `rand()` is called, you can put it before the `__init__` function (but still within this class) and reuse it everytime rand() is called